src/client/components/RightArrowComponent/RightArrowComponent.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './RightArrowComponent.styles.css';
+
+export default function RightArrowComponent(props) {
+  return (
+    <svg
+      onClick={props.clickRightArrow}
```

Comment:
It's not so good practice to add the clickhandler to the SVG itself. Better to add the clickhandler to `<a>` tag or a `<button>` to help screen readers and web crawlers navigate what to click on. Personally I would use a `<button>` since it better reflects that we will be updating some internal functionality in the app that will trigger some action, as opposed to "linking us out of the current page", but `<button>` and `<a>` are often used interchangeably. If you use button, remember to give it the attribute `type="button"`.

Reply:
Thank you for pointing it out. I will do the necessary changes.

Reply:
I have changed the logic of implementing the arrows by simply using html arrows. Please have a look.